test/broadcasting.jl

```diff
@@ -51,6 +51,9 @@ end
     @test (df1 .- df2) == DataFrame(zeros(size(refdf)), names(refdf))
     @test (df1 .* df2) == refdf .^ 2
     @test (df1 ./ df2) == DataFrame(ones(size(refdf)), names(refdf))
+
+    @test_throws ArgumentError DataFrame(a=1, b=1) .+ DataFrame(b=1, a=1)
+    @test DataFrame() .+ DataFrame() == DataFrame()
```

Comment:
This testset seems to be about non-`DataFrame` arguments, so better put this elsewhere?

Reply:
ok - moved